src/oatpp/core/data/mapping/type/Type.hpp

```diff
@@ -280,6 +280,7 @@ class Type {
        * Description.
        */
       std::string description = "";
+      std::string default_value = "";
```

Comment:
Please rename to `defaultValue`
Oat++ has a "non-standard" "legacy" code-style.

Reply:
My bad! True, everything is camel cased. I just fixed it.